src/cargo/util/interning.rs

```diff
@@ -13,7 +13,19 @@ use std::str;
 use std::sync::Mutex;
 use std::sync::OnceLock;
 
-static STRING_CACHE: OnceLock<Mutex<HashSet<&'static str>>> = OnceLock::new();
+pub static DEFAULT_INTERNED_STRING: InternedString = InternedString { inner: "default" };
```

Comment:
I'm not sure what the clearest naming pattern for this is. The current name suggests `Default::default()`. I am open to suggestions!

Reply:
`INTERNED_DEFAULT`?

Naming it for the content (and not bothering with the type/storage) or naming it with `DEFAULT` last helps to break that confusion.